server/src/models/BoardGame.js

```diff
@@ -12,32 +12,43 @@ class BoardGame extends unique(Model) {
   }
 
   static get relationMappings() {
-	const { Review } = require("./index")
-
-	return {
-		reviews: {
-			relation: Model.HasManyRelation,
-			modelClass: Review,
-			join: {
-				from: "boardGames.id",
-				to: "reviews.boardGameId"
-			}
+		const { Review } = require("./index")
+		const {User} = require("./index")
+
+
+		return {
+			reviews: {
+				relation: Model.HasManyRelation,
+				modelClass: Review,
+				join: {
+					from: "boardGames.id",
+					to: "reviews.boardGameId"
+				}
+			}, 
+			
+			user: {
+        relation: Model.BelongsToOneRelation,
+        modelClass: User,
+        join: {
+          from: "boardGame.userId",
+          to: "users.id"
+        }
+      }
 		}
 	}
-}
 
   static get jsonSchema() {
-	return {
-	type: "object",
-	  required: ["name", "minPlayers", "maxPlayers", "estimatedPlayTime", "description"],
-	  properties: {
-		name: { type: "string" },
-		minPlayers: { type: ["string", "integer"] },
-		maxPlayers: { type: ["string", "integer"] }, 
-		estimatedPlayTime: { type: ["string", "integer"] }, 
-		description: { type: "string" }
-	  }
-	}
+		return {
+			type: "object",
+				required: ["name", "minPlayers", "maxPlayers", "estimatedPlayTime", "description"],
+				properties: {
+				name: { type: "string" },
+				minPlayers: { type: ["string", "integer"] },
+				maxPlayers: { type: ["string", "integer"] }, 
+				estimatedPlayTime: { type: ["string", "integer"] }, 
+				description: { type: "string" }
+			}
+		}
```

Comment:
im not going to reject the PR for this, but its weird that this is the only file in this PR with extra spacing for tabs